### ISSUE TITLE
Getting errors when loading loom files

### ISSUE THREAD

**Issue by hawaiiki** [scverse/scanpy#247]
Hi,

I am getting an error when loading my loom files, which did not happen before and I am not capable of understanding the error output to try to fix it. 

![screen shot 2018-08-29 at 10 58 23](https://user-images.githubusercontent.com/42487820/44760841-9b527680-ab7b-11e8-9e85-0d0235cee6db.png)

Your help will be much appreciated.

**Comment by Koncopd** [comment]
Hi,

This happens because of the version of loompy, that is older than 2.0.10. It should work with 2.0.10.

**Comment by hawaiiki** [comment]
I upgraded loompy and scanpy as well but now I am getting an other error.

![screen shot 2018-08-29 at 19 21 34](https://user-images.githubusercontent.com/42487820/44782019-db881800-abc0-11e8-8948-90aa0b0c20a1.png)

![screen shot 2018-08-29 at 19 14 14](https://user-images.githubusercontent.com/42487820/44782040-eb076100-abc0-11e8-961e-75b4ba0c8ec7.png)

**Comment by falexwolf** [comment]
Apologies for the late response @hawaiiki! I married and moved to the US with twin babies last week. And in between, I spilled something over my laptop...

Yes, unfortunately, there were two half-cooked anndata releases out there. 😒 All these issues are fixed on GitHub and in anndata 0.6.10. anndata is now able to fully handle loom's layers, which it wasn't before and hence gained quite some additional functionality, thanks to @Koncopd and @VolkerBergen.
